README.md

```diff
@@ -48,6 +48,14 @@ SGI_IDS = Dict( "Rhone"       => "B43-03",
                 "Arolla"      => "B73-14",
                 "ArollaHaut"  => "B73-12")
 ```
+
+An alternative workflow is to first fetch the data for the entire Alps (the download step) and then select the geometry of a list of glacier given in the `SGI_IDS` dict automatically as following:
+```
+datadir = joinpath(@__DIR__,"mydata/")
+fetch_data(datadir)
+data = geom_select.(keys(SGI_IDS), values(SGI_IDS), datadir);
+```
+
```

Comment:
Is there a way to get a dict of all SGI-glaciers?  If so, would be good to document it here.

Reply:
Don't know. Maybe @christopheogier found something?